www/config.yml

```diff
@@ -27,6 +27,13 @@ chapters:
   facebook: pyladiesankara
   github: pyladiesankara
   website: ankara
+- email: freiburg@pyladies.com
+  location:
+    latitude: 47.997791
+    longitude: 7.842609
+  name: Freiburg, Germany
+  github: PyLadies-Freiburg
+  organizer : Hema
```

Comment:
Looks good except for one minor fix: a space after `organizer` needs to be deleted.